src/lib.rs

```diff
@@ -536,13 +717,13 @@ impl IntoRawFd for UnixStream {
 /// // close the listener socket
 /// drop(listener);
 /// ```
-pub struct UnixListener {
+pub struct UnixStreamListener {
```

Comment:
I think we should probably leave this as `UnixListener`. It's a bit of an unfortunate naming setup, but that's what the standard library setup has, and it's slated for stabilization.